README.md

```diff
@@ -90,10 +89,27 @@ end of the css file.
 
 Type: `Boolean`
 
-Render the banner all in one line
+Default: `false`
+
+Render the banner all in one line.
+
+NOTE: If the string contains new line chars, the resulting comment will
```

Comment:
I'm not following you here, sorry. If `inline` is not set to true, the banner will be rendered in multiline